### ISSUE TITLE
Timeout when using DNS name

### ISSUE THREAD

**Issue by pierrejochem** [duplicati/duplicati#1278]
I tried the Duplicati 2.0 Preview and got a problem with the resolution of DNS names. The same problem occurred with Version 1.2.4 but there I could solve it with the Public IP and set ignore untrusted certificates. We have an official certificate in the Server so no self signed.

![6d22e8bea0692e1af06db628c06fa04a26f1f31165de41d923 pimgpsh_fullsize_distr](https://cloud.githubusercontent.com/assets/9192953/6076637/8066376c-ade6-11e4-8e98-dac57559bc57.png)

**Comment by pierrejochem** [comment]
Nodody else run into this problem?

**Comment by renestach** [comment]
It seems like this error is happening when you test the connection to your server. Unfortunately, you do not say what kind of connection that is and what the hostname/URL looks like.

**Comment by pierrejochem** [comment]
exactly. We use the fqdn and can resolve this via ping but not via dublicati. 

*We are behind a proxy.* 

It is a https Url with a regular external confirmed certificate (goDaddy).
